crates/swc_bundler/src/bundler/chunk/cjs.rs

```diff
@@ -229,7 +229,7 @@ where
             return;
         }
 
-        if let ModuleItem::ModuleDecl(ModuleDecl::Import(i)) = node {
+        if let ModuleItem::ModuleDecl(box ModuleDecl::Import(i)) = node {
```

Comment:
Box patterns are going to be deprecated https://github.com/rust-lang/rust/issues/29641#issuecomment-1399430625

Reply:
I know, but refactoring will take much more time than this PR itself.

I filed https://github.com/swc-project/swc/issues/7039 instead

Reply:
Can I help on this one so that you don't go nightly?

Reply:
I think a follow-up PR would be better, because there's too many changes for me to review

Reply:
Anyway, thank you!
I was worried about the box_patterns work

Reply:
It's not very good timing because of the TypeScript 5.0 release upcoming.

Reply:
Going nightly for a perf change doesn't seem worthwhile.

Reply:
I did similar work several times, and always, the problem was that the previous branch should match everything from the if guard. Otherwise, it becomes a logic change.